README.md

```diff
@@ -70,82 +112,141 @@ Docker is needed to run the project in a containerized environment. Follow the a
 
 ### Setting Up Environment Variables
-Create or update the `.env` files in the `fastapi_backend` and `nextjs-frontend` directories with the following variables:
 
 **Backend (`fastapi_backend/.env`):**
-
-For local setup:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@localhost:5432/mydatabase
-```
-For Docker setup, replace localhost with db:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@db:5432/mydatabase
-```
-Other environment variables remain unchanged:
-
-```env
-ACCESS_SECRET_KEY=<your-secret-key>
-RESET_PASSWORD_SECRET_KEY=<your-secret-key>
-VERIFICATION_SECRET_KEY=<your-secret-key>
-OPENAPI_OUTPUT_FILE=../nextjs-frontend/openapi.json
-
-MAIL_USERNAME=<your-username>
-MAIL_PASSWORD=<your-password>
-MAIL_FROM=<your-from>
-MAIL_SERVER=<your-server>
+Copy the `.env.example` files to `.env` and update the variables with your own values.
+```bash
+cp project_name/.env.example project_name/.env
 ```
+1. You will only need to update the User Secrets. You can use the following command to generate a new secret key:
+   ```bash
+   python3 -c "import secrets; print(secrets.token_hex(32))"
+   ```
+2. The DATABASE, Email, OPENAPI, and FRONTEND_URL settings are ready to use locally.
+3. You can check the .env.example file for more information about the variables.
 
 **Frontend (`nextjs-frontend/.env.local`):**
-```env
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
-OPENAPI_OUTPUT_FILE=openapi.json
+Copy the `.env.example` files to `.env`. These values are unlikely to change, so you can leave them as they are.
+```bash
+cp project_name/.env.example project_name/.env
 ```
 
-### If you are not using Docker:
-To run the project locally, use the following commands:
-
-#### Backend
-1. Navigate to the `fastapi_backend` directory.
-2. Use Docker to run the database to avoid local installation issues. Build and start the database container:
+### Running the Database
+1. Use Docker to run the database to avoid local installation issues. Build and start the database container:
    ```bash
    docker compose build db
    docker compose up db
    ```
-3. Run the following command to apply database migrations:
+2. Run the following command to apply database migrations:
    ```bash
    make docker-migrate-db
    ```
-4. Start the FastAPI server:
+
+### Build the project (without Docker):
+To setup the project environment locally, use the following commands:
+
+#### Backend
+
+1. Navigate to the `fastapi_backend` directory and run:
    ```bash
-   make start-backend
+   poetry install
    ```
 
 #### Frontend
-1. Navigate to the `nextjs-frontend` directory.
-2. Install dependencies (if not already installed):
+1. Navigate to the `nextjs-frontend` directory and run:
+   ```bash
+   pnpm install
+   ```
+
+### Build the project (with Docker):
+
+1. Build the backend and frontend containers:
+   ```bash
+   make docker-build-backend
+   make docker-build-frontend
+   ```
+
+## Running the Application
```

Comment:
I don't think we should show start-backend if the recommended way to run is in hot-reload-mode. Just show the hot-reload mode.

Reply:
Now, the command is watch-backend and watch-frontend. What command name do you suggest?

Reply:
- make backend-hotreload
- make hot-backend
- make backend-live
- make backend-run-live

Reply:
There's a start-backend in the README. Check it.

Reply:
What I mean is: we should only recommend watch mode on readme. Not  start-backend.